pod.go

```diff
@@ -562,6 +562,10 @@ func createPod(podConfig PodConfig) (*Pod, error) {
 		return p, nil
 	}
 
```

Comment:
Please add a comment here to specify this part is only accessed when `createPod(podConfig PodConfig)` is called by `CreatePod()` from api.go, which basically means this is only called when we create the pod.